Modules/CaPTkCommon/cmdapps/CaPTkCohortJsonFromFilesystemCmdApp.cpp

```diff
@@ -0,0 +1,101 @@
+#include <mitkCommandLineParser.h>
+
+#include <CaPTkCohortJsonFromFilesystem.h>
+#include <CaPTkWriteJsonToFile.h>
+
+#include <json/json.h>
+
+#include <algorithm>
+#include <string>
+#include <fstream>
+#include <iostream>
+#include <memory>
+
+/**Splits a string into a list using a delimiter*/
+std::vector<std::string> 
+split(const std::string &s, char delim) {
```

Comment:
We should be using Qt's capability for all these tasks. Check [QString](https://doc.qt.io/qt-5/qstring.html#split)